### PR TITLE
Issue 4921 - FIX: tickets table slide not loading and custom status is missing

### LINKED ISSUE #4921: Tickets forever loading in the tickets table
### Description
With a particular template the ticket never finishes loading in tickets template

- This is happening in: staging
- This requires a hotfix to production: no

### Steps to replicate
- Create a ticket with the following  template:
<details>
<summary>Immutable template </summary>

```
{
  "modules": [
    {
      "properties": [
        {
          "type": "boolean",
          "default": false,
          "immutable": true,
          "name": "Boolean"
        },
        {
          "type": "text",
          "immutable": true,
          "name": "Text"
        },
        {
          "type": "longText",
          "immutable": true,
          "name": "Long Text"
        },
        {
          "type": "date",
          "immutable": true,
          "name": "Date"
        },
        {
          "type": "number",
          "immutable": true,
          "name": "Number"
        },
        {
          "type": "image",
          "immutable": true,
          "name": "Image"
        },
        {
          "type": "coords",
          "immutable": true,
          "name": "Coordinates"
        },
        {
          "type": "oneOf",
          "immutable": true,
          "values": "jobsAndUsers",
          "name": "One of"
        },
        {
          "type": "manyOf",
          "immutable": true,
          "values": "riskCategories",
          "name": "Many of"
        }
      ],
      "name": "immutable properties"
    },
    {
      "type": "safetibase",
      "properties": [
        {
          "type": "boolean",
          "default": false,
          "name": "Risk Assessed"
        }
      ]
    },
    {
      "properties": [
        {
          "type": "coords",
          "name": "Colour Undefined"
        },
        {
          "type": "coords",
          "color": [
            255,
            0,
            0
          ],
          "name": "Colour Specified (red)"
        },
        {
          "type": "boolean",
          "default": false,
          "name": "Boolean"
        },
        {
          "type": "coords",
          "name": "Colour By Property (Switch)",
          "color": {
            "property": {
              "module": "Coloured Pins",
              "name": "Boolean"
            },
            "mapping": [
              {
                "default": [
                  245,
                  245,
                  220
                ]
              },
              {
                "value": true,
                "color": [
                  160,
                  255,
                  0
                ]
              }
            ]
          }
        },
        {
          "type": "coords",
          "color": {
            "property": {
              "name": "Status"
            },
            "mapping": [
              {
                "default": [
                  188,
                  190,
                  202
                ]
              },
              {
                "value": "Open",
                "color": [
                  107,
                  119,
                  140
                ]
              },
              {
                "value": "In Progress",
                "color": [
                  113,
                  86,
                  255
                ]
              },
              {
                "value": "For Approval",
                "color": [
                  2,
                  136,
                  209
                ]
              },
              {
                "value": "Closed",
                "color": [
                  46,
                  125,
                  50
                ]
              },
              {
                "value": "Void",
                "color": [
                  0,
                  0,
                  0
                ]
              }
            ]
          },
          "name": "Colour By Property (Status)"
        },
        {
          "type": "text",
          "name": "Colour Textbox"
        },
        {
          "type": "coords",
          "color": {
            "property": {
              "module": "Coloured Pins",
              "name": "Colour Textbox"
            },
            "mapping": [
              {
                "default": [
                  255,
                  255,
                  255
                ]
              },
              {
                "value": "black",
                "color": [
                  0,
                  0,
                  0
                ]
              },
              {
                "value": "red",
                "color": [
                  255,
                  0,
                  0
                ]
              },
              {
                "value": "green",
                "color": [
                  0,
                  255,
                  0
                ]
              },
              {
                "value": "blue",
                "color": [
                  0,
                  0,
                  255
                ]
              },
              {
                "value": "hotpink",
                "color": [
                  255,
                  105,
                  180
                ]
              }
            ]
          },
          "name": "Colour By Property (Textbox)"
        },
        {
          "name": "Colour Threshold Test",
          "type": "oneOf",
          "values": [
            "PRIMARY_CONTRAST",
            "TERTIARY_LIGHTEST",
            "TERTIARY_LIGHTER",
            "BASE_LIGHTEST",
            "BASE_LIGHTER",
            "BASE_LIGHT",
            "ERROR_LIGHTEST",
            "BLACK"
          ]
        },
        {
          "name": "Colour by Property (Select)",
          "type": "coords",
          "color": {
            "property": {
              "module": "Coloured Pins",
              "name": "Colour Threshold Test"
            },
            "mapping": [
              {
                "default": [
                  250,
                  229,
                  154
                ]
              },
              {
                "value": "PRIMARY_CONTRAST",
                "color": [
                  255,
                  255,
                  255
                ]
              },
              {
                "value": "TERTIARY_LIGHTEST",
                "color": [
                  242,
                  246,
                  252
                ]
              },
              {
                "value": "TERTIARY_LIGHTER",
                "color": [
                  232,
                  238,
                  248
                ]
              },
              {
                "value": "BASE_LIGHTEST",
                "color": [
                  208,
                  217,
                  235
                ]
              },
              {
                "value": "BASE_LIGHTER",
                "color": [
                  193,
                  200,
                  213
                ]
              },
              {
                "value": "BASE_LIGHT",
                "color": [
                  188,
                  190,
                  202
                ]
              },
              {
                "value": "ERROR_LIGHTEST",
                "color": [
                  249,
                  236,
                  236
                ]
              },
              {
                "value": "BLACK",
                "color": [
                  0,
                  0,
                  0
                ]
              }
            ]
          }
        }
      ],
      "name": "Coloured Pins"
    }
  ],
  "properties": [
    {
      "name": "Colour by Property (Priority)",
      "type": "coords",
      "color": {
        "property": {
          "name": "Priority"
        },
        "mapping": [
          {
            "default": [
              107,
              119,
              140
            ]
          },
          {
            "value": "Low",
            "color": [
              2,
              136,
              209
            ]
          },
          {
            "value": "Medium",
            "color": [
              237,
              108,
              2
            ]
          },
          {
            "value": "High",
            "color": [
              190,
              67,
              67
            ]
          }
        ]
      }
    }
  ],
  "config": {
    "comments": true,
    "issueProperties": true,
    "pin": {
      "color": {
        "property": {
          "module": "safetibase",
          "name": "Risk Assessed"
        },
        "mapping": [
          {
            "default": [
              235,
              0,
              0
            ]
          },
          {
            "value": true,
            "color": [
              0,
              235,
              0
            ]
          }
        ]
      }
    },
    "status": {
      "values": [
        {
          "name": "Awaiting triage",
          "type": "open"
        },
        {
          "name": "Triaged (This option is extra long to test for overflow)",
          "type": "open"
        },
        {
          "name": "Active",
          "type": "active"
        },
        {
          "name": "To verify",
          "type": "review"
        },
        {
          "name": "Approved",
          "type": "done"
        },
        {
          "name": "Won't fix",
          "type": "void"
        }
      ],
      "default": "Awaiting triage"
    }
  },
  "code": "IMM",
  "name": "Immutable Template"
}
```
</details>

- Go into the tickets table
- Click on the created ticket

### Current Behaviour
- The ticket never finishes loading

### Expected Behaviour
- It should load the ticket and show the properties

### Pictures/ Gifs of the error
image

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #4921 

#### Description
Template data was not fetched, this was fixed.
There was also a problem with the status chip for custom status that was empty, that has also been fixed

#### Test cases
Use the template provided in the issue description and check you can access a ticket slide without it loading forever.
Check also the status chip is always displayed

### CODE REVIEW

File: frontend/src/v5/store/projects/projects.selectors.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ import { createSelector } from 'reselect';
 import { selectCurrentTeamspace } from '../teamspaces/teamspaces.selectors';
 import { IProjectsState } from './projects.redux';
 import { IProject } from './projects.types';
+import { DEFAULT_STATUS_CONFIG } from '@controls/chip/chip.types';
```
Comment (sanmont3drepo) [inline frontend/src/v5/store/projects/projects.selectors.ts:22]: Status is not showing when opening the ticket
image

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketsTopPanel/statusProperty/statusProperty.component.tsx [hunk 2]
```diff
@@ -38,7 +38,9 @@ export const StatusProperty = ({ onBlur, readOnly }: StatusPropertyProps) => {
 	const ticketView = TicketsCardHooksSelectors.selectView();
 	const templateId = ticketView === TicketsCardViews.New ? templateIdNewTicket : templateIdExistingTicket ?? templateIdTabularView;
 
-	const statusConfig = TicketsHooksSelectors.selectStatusConfigByTemplateId(containerOrFederation, templateId);
+	const statusConfig = templateIdTabularView
+		? ProjectsHooksSelectors.selectStatusConfigByTemplateId(templateId)
+		: TicketsHooksSelectors.selectStatusConfigByTemplateId(containerOrFederation, templateId);
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketsTopPanel/statusProperty/statusProperty.component.tsx:43]: can we change TicketsHooksSelectors.selectStatusConfigByTemplateId so it also takes information from the projecttemplates?

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

**Review by The-Daniel — APPROVED** [review]

**Review by sanmont3drepo — APPROVED** [review]
